notices.json

```diff
@@ -1,4 +1,41 @@
 [
+      {
+    "id": "1",
```

Comment:
I rebased this one to a new PR here against the `release` branch: https://github.com/mattermost/notices/pull/220